output/config.yaml

```diff
@@ -0,0 +1,43 @@
+# Configuration from TopoStats run completed : 2022-08-09 11:09:18
```

Comment:
Don't think this file should be included as its part of the output.

Reply:
Yep absolutely! Don't remember how it snuck in. Resolved in f092894